paths.go

```diff
@@ -86,8 +86,11 @@ func (p *Paths) initAstilectronDirectory() {
 	}
 }
 
-// initElectronDownloadSrc initializes the electron download source path
-func (p *Paths) initElectronDownloadSrc(os, arch string) {
+func AstilectronDownloadSrc() string {
+	return fmt.Sprintf("https://github.com/asticode/astilectron/archive/v%s.zip", VersionAstilectron)
+}
+
+func ElectronDownloadSrc(os, arch string) string {
```

Comment:
Can you add a comment on top of this function of the form `// ElectronDownloadSrc`?